README.md

```diff
@@ -363,6 +363,48 @@ var_dump($flat);
 // {0: 'one', 1: 'two', 0: 'three'}
 ```
 
+## More examples
+
+With the example data above, this is how you could use itertools to get all unique colors of the cars in alphabetical order:
+
+```php
+use function Zicht\Itertools\iterable;
+
+$vehicles =iterable($vehicles)
+    ->filter(equals('car', 'type')) // {[vehicle...], [vehicle...]}
+    ->map('colors') // {0: ['red', 'green', 'blue'], 1: ['blue']}
+    ->collapse() // {0: 'red', 1: 'green', 2: 'blue', 3: 'blue'}
+    ->unique() // {0: 'red', 1: 'green', 2: 'blue'}
+    ->sorted(); // {2: 'blue', 1: 'green', 0: 'red'}
+```
+
+
+Assuming the variable `vehicles` is defined in the controller that calls the template, you can achieve the same in Twig.
```

Comment:
I think we should just assume that `vehicles` is available, i.e. this text could be reduced to `You can achieve the same in Twig:`